onebusaway-android/src/main/java/org/onebusaway/android/util/UIUtils.java

```diff
@@ -1141,4 +1138,19 @@ public static String formatTime(Context context, long time) {
                         DateUtils.FORMAT_NO_MIDNIGHT
         );
     }
+
+    /**
+     * Set smaller text size if the route short name has more than 3 characters
+     *
+     * @param view Text view
+     * @param routeShortName Route short name
+     */
+    public static void setTextSizeForArrivalsListAdapterStyleARoute(TextView view,
```

Comment:
Let's rename this method to `maybeShrinkLongRouteName(TextView view, String routeShortName)`.  And, let's make the first line:
~~~
if (routeShortName.length() < 4) {
    return;  // No-op if text is short enough to fit
} else if (...
~~~
I think this makes it clearer that this method doesn't always modify the view/text.